test/benchmarks/content.read.js

```diff
@@ -121,11 +121,11 @@ module.exports = (suite, CACHE) => {
     },
     fn (deferred) {
       const stream = read.stream(CACHE, INTEGRITY)
-      stream.on('data', () => {})
-      stream.on('error', (err) => deferred.reject(err))
-      stream.on('end', () => {
-        deferred.resolve()
-      })
+      stream.promise().then(
+        () => deferred.resolve(),
+        er => deferred.reject(er)
+      )
+      stream.resume()
```

Comment:
unrelated to these specific lines but reading through these made me realize that we're actually slightly changing the exposed public APIs since the returned streams are now `minipass` streams and might have [differences from Node.js streams](https://www.npmjs.com/package/minipass#differences-from-nodejs-streams) - I wonder if it's worth mentioning something in the `README` file

Reply:
Yeah, doc update would be good.  As it was, Cacache returned a mix of various mississippi streams, but most of those inherited from stream.Readable or stream.Duplex, so they would function more or less identically to node-core streams.

Reply:
Agreed.